flink-python/src/main/java/org/apache/flink/python/util/ResourceUtil.java

```diff
@@ -22,71 +22,45 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.List;
 
 /**
  * Utils for building the most basic environment for running python udf workers. The basic environment does not include
  * python part of Apache Beam. Users need to prepare it themselves.
  */
 public class ResourceUtil {
 
-	public static final String[] BUILT_IN_PYTHON_DEPENDENCIES = {
-		"pyflink.zip",
-		"py4j-0.10.8.1-src.zip",
-		"cloudpickle-1.2.2-src.zip",
-		"pyflink-udf-runner.sh"
-	};
+	public static final String PYFLINK_UDF_RUNNER = "pyflink-udf-runner.sh";
```

Comment:
Should we move pyflink-udf-runner.sh to src/main/resources of the jar of flink-python then I think there is no necessary to extract it from the jar file any more?